CONTRIBUTING.md

```diff
@@ -164,6 +164,16 @@ To keep React UI consistent and predictable the following guidelines should be o
 
 1. If component accepts the `children` prop it should be either required or the element
    should return `null` when no children are provided.
+2. When forwarding HTML attributes to the component the following rules should
```

Comment:
We still need to deal with `Modal` and `Radio` components. See issue: https://github.com/react-ui-org/react-ui/issues/496